Bottom: adba93d8c8cedf2859812ffc355a69d314a590d4
Top:    422dfd4290eef95e33f27c929af3fe2178f0dc09
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-10-12 13:34:28 +0800

Refresh of netfilter-add-a-netbuffer

---

diff --git a/net/filter-buffer.c b/net/filter-buffer.c
index 336bdcc..57be149 100644
--- a/net/filter-buffer.c
+++ b/net/filter-buffer.c
@@ -46,7 +46,7 @@ static void filter_buffer_release_timer(void *opaque)
     /*
      * Note: filter_buffer_flush() drops packets that can't be sent
      * TODO: We should leave them queued.  But currently there's no way
-     * for the next filter or recivier to notify us that it can receive
+     * for the next filter or receiver to notify us that it can receive
      * more packets.
      */
     filter_buffer_flush(nf);
